package.json

```diff
@@ -28,9 +28,7 @@
   },
   "homepage": "https://github.com/rpp34-boc-victoria/Trim.io#readme",
   "dependencies": {
-    "@emotion/react": "^11.10.0",
-    "@emotion/styled": "^11.10.0",
-    "@mui/material": "^5.9.3",
```

Comment:
Have you made sure these can be removed without affecting anyone else's component?

Reply:
idk y this been removed. lol